protocol/communities/community.go

```diff
@@ -269,6 +269,11 @@ func (o *Community) initialize() {
 	}
 }
 
+type CommunitySettings struct {
+	CommunityID                  string `json:"communityId"`
+	HistoryArchiveSupportEnabled bool   `json:"historyArchiveSupportEnabled"`
+}
```

Comment:
I think I remember you saying that in the future the `HistoryArchiveSupportEnabled ` would eventually be split into fields that map individually to `message_archive_seeding_enabled ` and `message_archive_fetching_enabled `. So at the moment both these settings will be either true together or false together?